Sources/FoundationEssentials/CMakeLists.txt

```diff
@@ -99,7 +99,7 @@ if(NOT BUILD_SHARED_LIBS)
     target_compile_options(FoundationEssentials PRIVATE
         "SHELL:$<$<COMPILE_LANGUAGE:Swift>:-Xfrontend -public-autolink-library -Xfrontend $<$<PLATFORM_ID:Windows>:${CMAKE_STATIC_LIBRARY_PREFIX_Swift}>_FoundationCollections>")
     target_compile_options(FoundationEssentials PRIVATE
-        "SHELL:$<$<COMPILE_LANGUAGE:Swift>:-Xfrontend -public-autolink-library -Xfrontend swiftSynchronization>")
+        "SHELL:$<$<COMPILE_LANGUAGE:Swift>:-Xfrontend -public-autolink-library -Xfrontend $<$<PLATFORM_ID:Windows>:${CMAKE_STATIC_LIBRARY_PREFIX_Swift}>swiftSynchronization>")
```

Comment:
Is this missing from the `_FoundationCShims` option above as well?

Reply:
No, it isn't present on `_FoundationCShims` since that is a C library, and we do not currently touch the C library naming on Windows. The Swift libraries are the only ones that we are currently prefixing (in the hopes that we can merge the static and shared directories some day).